module/damage/applydamage.js

```diff
@@ -514,12 +514,11 @@ export default class ApplyDamageDialog extends Application {
             ? 'HT'
             : effect.modifier < 0
               ? `HT+${-effect.modifier}`
-              : `HT-${effect.modifier}`
-          (effect?.modifier ?? 0) === 0
-            ? 'HT'
-            : effect.modifier < 0
-              ? `HT+${-effect.modifier}`
-              : `HT-${effect.modifier}`
+              : `HT-${effect.modifier}`(effect?.modifier ?? 0) === 0
```

Comment:
On second look, this code is wrong.  It looks like a copy-paste error -- everything after ```: `HT-${effect.modifier}``` on this line is a repeat of lines 513-517 and should be removed.

Reply:
Donezo. I thought it looked off.